web/components/utils.js

```diff
@@ -0,0 +1,5 @@
+// contains helper functions shared by component library
+
+const concatClassName = (baseClassName, extraClassName = undefined) => (extraClassName ? [baseClassName, extraClassName].join(' ') : baseClassName);
```

Comment:
this seems like a pretty useless function - why not just use a template string?

```js
className={`defaultClass ${className}`}
```

avoids the unnecessary import

Reply:
```className={`defaultClass ${className || sharedClassName}`}``` resulted in `undefined` being in the class name when neither variables were defined (even adding `|| ""` to the end of the ternary didn't work). I'll go ahead and remove this since I'll be removing `sharedClassName` altogether.

Reply:
Since convention is to destructure prop args straight off the bat, you can provide defaults:

```js
const myComponent = ({ className = '' }) => (< ... />)
```

I might have forgotten to do this though, I'll add it if I see it

Reply:
ooh good catch, we should handle `className` being undefined

Reply:
we could probably enforce this by setting `className` as required with `propTypes`